crates/oxc_cli/src/lib.rs

```diff
@@ -1,9 +1,10 @@
 mod command;
 mod walk;
 
-use std::{fs, path::Path, rc::Rc};
+use std::{fs, path::Path, path::PathBuf, rc::Rc};
 
 pub use command::Command;
+use miette::Report;
```

Comment:
I exported an alias from `oxc_diagnostics`, just in case I change this one day.

Reply:
How should I import this? I was hoping it'd be:
```rust
use oxc_diagnostics::Report;
```
but that doesnt seem to work 😅

Reply:
Im getting: 
<img width="640" alt="image" src="https://user-images.githubusercontent.com/17054057/221369731-afe5ab8e-7e58-40a5-aa3a-486dd531b9f3.png">

Reply:
Add it to the crate's Cargo.toml

Reply:
ahh, that did the trick, should be fixed now :)